utils/translation-gpt/index.mjs

```diff
@@ -0,0 +1,120 @@
+import {readFileSync, writeFileSync} from 'fs';
+import {env} from 'process';
+
+const fileToTranslate = JSON.parse(
+  readFileSync('../../packages/atomic/src/locales.json')
+);
+const fileAlreadyTranslated = JSON.parse(readFileSync('./temporary.json'));
+
+const allLanguages = [
+  'en',
+  'fr',
+  'cs',
+  'da',
+  'de',
+  'el',
+  'es',
+  'fi',
+  'hu',
+  'id',
+  'it',
+  'ja',
+  'ko',
+  'nl',
+  'no',
+  'pl',
+  'pt',
+  'pt-br',
+  'ru',
+  'sv',
+  'th',
+  'tr',
+  'zh',
+  'zh-cn',
+  'zh-tw',
+];
+
+const prompt = (mainKey, toTranslate, languages) =>
+  `I will give you a string to translate. The string can possible contain variables between braces similar to this: {{variable}}. 
+  Those variables are meant to represent dynamic values.
+  If there are no variable, then do a normal translation.
+  I want you to translate it in the language code ${languages.join(
+    ', '
+  )} and to place the variable with braces correctly in the new string if it applies.
+  For each language code, answer in valid JSON format, with the the main key being ${mainKey} and each individual language code is a property of ${mainKey}. 
+  The value of each property of ${mainKey} should be the translated string.
+  The string to translate is: 
+  "${toTranslate}"`;
+
+async function main() {
+  for (const translationKey of Object.keys(fileToTranslate)) {
+    if (fileAlreadyTranslated[translationKey]) {
```

Comment:
What happens if we want to add (or delete) a language to the list of supported languages?

Reply:
- Add: Rerun the script, it will ask GPT for the new language for each entry in `locales.json`
- Delete: Nothing